includes/class-wc-gateway-dintero-hp.php

```diff
@@ -1201,13 +1195,6 @@ private function check_capture( $order_id ) {
 			}
 			$transaction = self::_adapter()->get_transaction($transaction_id);
 
-
-			if ( ! array_key_exists( 'merchant_reference', $transaction ) ) {
-				$order->add_order_note(__('Could not capture transaction: Merchant reference id not found'));
-				$order->save_meta_data();
-				return false;
-			}
-
```

Comment:
should we not add a note if capture is "aborted" because of missing merchant reference?

Reply:
do we need to check the merchant_reference when the transaction is resolved from the order?
https://github.com/Dintero/Dintero.Checkout.WooCommerce/pull/96/files#diff-4aaca69ad3c4f3a1983beefb078889a21699f59e2f8473dbe000e11e4f594248L1215